aws/app/lambda/reliability/lib/dataLayer.js

```diff
@@ -20,25 +20,22 @@ async function getSubmission(message) {
   return await db.send(new GetItemCommand(DBParams));
 }
 
-async function saveToVault(submissionID, formSubmission, formID, language, createdAt) {
+async function saveToVault(submissionID, formResponse, formID, language, createdAt, securityAttribute) {
   const db = new DynamoDBClient({ region: REGION, endpoint: process.env.AWS_SAM_LOCAL ? "http://host.docker.internal:4566": undefined });
-  //get files from submission object.
-  let formSubmissionData = formSubmission.responses ?? formSubmission;
```

Comment:
Why was this line removed ?

Reply:
We  don't need to check `formSubmission.responses` anymore  the `responses` are passed directly to the saveToVault. So the line was removed.